protocol/x/clob/keeper/liquidations.go

```diff
@@ -452,6 +452,28 @@ func (k Keeper) GetFillablePrice(
 		return nil, err
 	}
 
+	// stat liquidation order for negative TNC when in DeliverTx
+	if tncBig.Sign() < 0 && !ctx.IsCheckTx() && !ctx.IsReCheckTx() {
+		telemetry.IncrCounterWithLabels(
+			[]string{metrics.Liquidations, metrics.LiquidationMatchNegativeTNC},
+			1,
+			[]gometrics.Label{
+				metrics.GetLabelForStringValue(
+					metrics.Callback,
+					metrics.DeliverTx,
+				),
+				metrics.GetLabelForStringValue(
```

Comment:
subaccount owner cardinality is infinite, but this is only emitted for subaccounts with negative TNC. Don't imagine this metric will be emitted very frequently and when it is it would be helpful to have this information IMO

Reply:
LMK if you think I should remove though

Reply:
maybe I can leave it in for this but for the liquidation notional quote quantums stat I can leave it out. That metric will be emitted frequently